CMakeLists.txt

```diff
@@ -297,6 +299,7 @@ if(Boost_FOUND)
 
   add_test(NAME IntegrationTest
     COMMAND IntegrationTest --log_level=test_suite --color_output=true -- "$<TARGET_FILE:bread>" "$<TARGET_FILE_DIR:IntegrationTest>" "${PROJECT_SOURCE_DIR}")
+  set_property(TEST IntegrationTest PROPERTY ENVIRONMENT ASAN_OPTIONS=detect_leaks=1)
```

Comment:
https://kristerw.blogspot.com/2018/06/useful-gcc-address-sanitizer-checks-not.html

Reply:
https://github.com/Morgan-Stanley/binlog/issues/46